setup.py

```diff
@@ -33,7 +33,7 @@ def unique_flatten_dict(d):
     'networkx': ['networkx>=2.5'],
     'gremlin': ['gremlinpython'],
     'bolt': ['neo4j', 'neotime'],
-    'nodexl': ['openpyxl', 'xlrd'],
+    'nodexl': ['openpyxl==3.1.0', 'xlrd'],
```

Comment:
I'm guessing should be something like `openpyxl<3.1.1` ? Better, is there a way to say `openpyxl<3.1.1 OR openpyxl > 3.1.1` ?

Reply:
Another option is leave unpinned here, and do the pinning as a fix at the `gha` level?

Reply:
fixed